src/components/Amalgamation/AmalgamatingBusinesses.vue

```diff
@@ -567,7 +567,7 @@ export default class AmalgamatingBusinesses extends Mixins(AmalgamationMixin, Co
 
       // fetch the new holding/primary business' data and update the prepopulated data
       // this will overwrite office addresses, directors, share structure and contact info
-      this.updatePrepopulatedData(business, true)
+      await this.updatePrepopulatedData(business, true)
```

Comment:
Without this, the code continued to line 576.

Instead, we needed to wait for updatePrepopulatedData() to finish and possibly throw an error that would be caught by line 571.